c++/src/H5DataSpace.h

```diff
@@ -43,7 +43,7 @@ class H5_DLLCPP DataSpace : public IdComponent {
     DataSpace &operator=(const DataSpace &rhs);
 
     // Closes this dataspace.
-    virtual void close();
+    virtual void close() override;
```

Comment:
Since you've defined the H5_OVERRIDE macro, do you want to use it in the code?

Reply:
Yes. But as I said in https://github.com/HDFGroup/hdf5/issues/334 `H5Cpp.h` doesn't work for me, it seems to not be included by any of the headers where I added override. In fact these headers don't have any `#include` statements at all, so I'm a bit confused where to put `H5_OVERRIDE`.

Reply:
OK, I defined `H5_OVERRIDE` in `H5Include.h`.  Seems like a correct enough place.  There's a similar block there for ancient C++ that has no `bool` keyword (!).